tests/director/api/tenants_api_test.go

```diff
@@ -17,17 +17,17 @@ func TestQueryTenants(t *testing.T) {
 	ctx := context.Background()
 
 	getTenantsRequest := fixTenantsRequest()
-	output := []*graphql.Tenant{}
+	var output []*graphql.Tenant
 	defaultTenants := fixDefaultTenants()
 
 	// WHEN
-	t.Log("List tenants")
+	t.Log("List tenant")
 	err := tc.RunOperationWithoutTenant(ctx, getTenantsRequest, &output)
 	require.NoError(t, err)
 
 	//THEN
 	t.Log("Check if tenants were received")
-	assert.Equal(t, 6, len(output))
+	assert.Equal(t, len(defaultTenants), len(output))
```

Comment:
this assertion is too strict, because someone can define their own tenants, we should check only if our test tenants were returned